docs/platforms/emr_serverless.md

```diff
@@ -1,61 +1,129 @@
 # AWS EMR Serverless Spark
 Amazon EMR (Elastic MapReduce) Serverless is a deployment option for Amazon EMR that provides a serverless runtime environment. This simplifies the operation of analytics applications that use the latest open-source frameworks, such as Apache Spark and Apache Hive. With EMR Serverless, you don’t have to configure, optimize, secure, or operate clusters to run applications with these frameworks.
 
-Bruin supports EMR Serverless as a spark orchestration platform. You can use bruin to integrate your spark workloads into complex pipelines that different data technologies, all without leaving your terminal. 
+Bruin supports EMR Serverless as a data platform. You can use bruin to integrate your spark workloads into complex pipelines that use different data technologies, all without leaving your terminal. 
 
 ## Connection
 
-In order to use bruin to trigger spark jobs in EMR Serverless, you need to define an `aws` connection in your `.bruin.yml` file. The connection schema looks like the following:
-```yaml
-        connections:
-            aws:
-            - name: aws-connection
-              access_key: _YOUR_AWS_ACCESS_KEY_ID_
-              secret_key: _YOUR_AWS_SECRET_ACCESS_KEY_
+In order to use bruin to run spark jobs in EMR Serverless, you need to define an `emr_serverless` connection in your `.bruin.yml` file. Here's a sample `.bruin.yml` with the required fields defined.
+
+```yaml 
+environments:
+  default:
+    connections:
+      emr_serverless:
+      - name: emr_serverless-default
+        access_key: AWS_ACCESS_KEY_ID
+        secret_key: AWS_SECRET_ACCESS_KEY
+        application_id: EMR_APPLICATION_ID
+        execution_role: IAM_ROLE_ARN
+        region: eu-north-1
+
+```
+
+## Logs
+Bruin supports log streaming for spark jobs. This is only supported for spark logs stored in [S3](https://docs.aws.amazon.com/emr/latest/EMR-Serverless-UserGuide/logging.html#jobs-log-storage-s3-buckets). Both `DRIVER` and `EXECUTOR` logs are streamed by default.
+
+In order to stream logs, one of the following conditions must be met:
+* Your EMR Serverless Application is pre-configured with an S3 Log Storage Location. 
+* `parameters.logs` or `parameters.workspace` is be specified and points to an S3 URI.
+
+## Asset Types
+
+Bruin supports two different ways of defining a spark asset
+
+### Python Asset
+A fully managed option where bruin takes care of job setup, configuration, and execution. You only need to define the workload logic.
```

Comment:
```suggestion
A fully managed option where Bruin takes care of job setup, configuration, and execution. You only need to define the workload logic.
```

Reply:
It wouldn't hurt to explain a bit more what exactly Bruin does in this option in a technical sense.